extmod/ntptime_py/ntptime/__init__.py

```diff
@@ -6,9 +6,11 @@
     import ustruct as struct
 except:
     import struct
-
-# (date(2000, 1, 1) - date(1900, 1, 1)).days * 24*60*60
-NTP_DELTA = 3155673600
+try:
+    # The epoch_override module will only be present on builds where the epoch year is not 2000
+    from ntptime.epoch_override import EPOCH_YEAR
+except:
+    EPOCH_YEAR = 2000
```

Comment:
It's possible to detect the Epoch used by a system at runtime using `time.gmtime(0)[0]`.  That's the year of the Epoch (1970 or 2000).  I think it'll be much simpler to use that mechanism to detect it.

Reply:
Why didn't I think of that?! 🙈
Thank you, this makes it [**so** much simpler](https://github.com/micropython/micropython/blob/4b4843f45eed6d63a70b83a009e7c049c7602361/extmod/ntptime.py).